ticket.rb

```diff
@@ -0,0 +1,21 @@
+require 'standard_item'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.